framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -1516,7 +1563,10 @@ public AnnotationMirror convertIntValToIntRange(AnnotationMirror intValAnno) {
         return createIntRangeAnnotation(Collections.min(intValues), Collections.max(intValues));
     }
 
-    /** Returns a {@code Range} bounded by the values specified in the given annotation. */
+    /**
+     * Returns a {@code Range} bounded by the values specified in the given {@code @Range}
```

Comment:
Should the `@Range` be `@IntRange`?
What about `@IntRangeFromPositive`?
What annotation is covered by the else branch?
Instead of the else branch, why not make the else code top level? (The earlier two ifs also don't use elses.)